doc/_static/tape_mpl/tape_mpl_examples.py

```diff
@@ -80,7 +80,7 @@ def rcparams(tape):
 
     plt.savefig(folder / "rcparams.png")
     plt.close()
-    plt.style.use("default")
+    use_drawer_style("black_white")
```

Comment:
All of the functions called after `rcparams` are affected by the updates to `plt.rcParams`, which causes the lines to be bolder in `wire_labels.png`, `postprocessing.png`, `mid_measure.png` in the `doc/_static/tape_mpl` directory.

Reply:
Images just had to be re-rendered.